code/modules/mob/mob.dm

```diff
@@ -609,16 +609,22 @@ note dizziness decrements automatically in the mob's Life() proc.
 	is_dizzy = 1
 	while(dizziness > 100)
 		if(client)
+			if(buckled || resting)
+				client.pixel_x = 0
+				client.pixel_y = 0
+		else
```

Comment:
The else should be an else for `if(buckled || resting)`; not if theres no client. Everything that is client.whatever needs a client to be valid.

Reply:
whoops. fixed